ckanext/dcatapchharvest/tests/test_dcatap_ch_serialize.py

```diff
@@ -78,42 +79,46 @@ def test_graph_from_dataset(self):
             distribution = URIRef(dh.resource_uri(resource_dict))
             assert self._triple(g, distribution, RDF.type, DCAT.Distribution)
             for link in resource_dict.get("documentation", []):
-                assert self._triple(g, distribution, FOAF.page, URIRef(link))   
-            
-            #e2c50e70-67ad-4f86-bb1b-3f93867eadaa    
+                assert self._triple(g, distribution, FOAF.page, URIRef(link))
+
+            eq_(
+                len([t for t in g.triples((distribution, DCAT.accessService, None))]),
+                len(resource_dict.get("access_services", []))
+            )
+            for link in resource_dict.get("access_services", []):
```

Comment:
since we are using `resource_dict.get("access_services", [])` two times already 
i would define them firstly like: access_services = resource_dict.get("access_services", [])

and then use it here and in the line 86